CHANGELOG.md

```diff
@@ -4,6 +4,49 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.18](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.18) - 2021-10-05
+
```

Comment:
We might be missing a general description here.

Reply:
```suggestion

Zebra's latest alpha updates dependencies and consensus parameters for NU5 testnet activation, and continues work on the mempool. 
```